Makefile

```diff
@@ -80,6 +80,7 @@ GOARCH ?= amd64
 LDFLAGS := $(shell cat hack/make/ldflags.txt)
 LDFLAGS_CSI := $(LDFLAGS) -X "$(MOD_NAME)/pkg/csi/service.Version=$(VERSION)"
 LDFLAGS_SYNCER := $(LDFLAGS) -X "$(MOD_NAME)/pkg/syncer.Version=$(VERSION)"
+LDFLAGS_CNSCTL := $(LDFLAGS) -X "main.Version=$(VERSION)"
```

Comment:
Shouldn't `cnsctl` version be independent of driver & syncer versions? Printing the version as something like `v2.1.0-rc.1-<sha>` doesn't look user friendly IMO.

```
root@ubuntu:~/github-csi2/src/sigs.k8s.io/vsphere-csi-driver# cnsctl.linux_amd64 -v
cnsctl version v2.1.0-rc.1-358-g65c54631-dirty
```

Reply:
>Shouldn't cnsctl version be independent of driver & syncer versions?

No, it gets harder to track the releases. The plan is to release this along with the driver and syncer and the tool will get the same version. 

>Printing the version as something like v2.1.0-rc.1-<sha> doesn't look user friendly IMO.

This applies for driver and syncer too. When the tool is released, the version with match the release tag.